src/plugins/index.js

```diff
@@ -57,9 +60,15 @@ export function TextContent(options={}) {
         'subscript',
     ]
 
+    const preformat = {
+        inlines: ['code', 'docref', 'link', 'term', 'xref'],
+        ...options.preformat || {},
+    }
+
     return [
         Code(options.code),
         Footnote({ marks }),
+        Preformat({ marks, preformat }),
```

Comment:
`Preformat` takes an object with `marks` and `inlines` properties, not `marks` and `preformat`. You've probably meant 

```suggestion
        Preformat({ marks, ...preformat }),
```